src/cts/README.md

```diff
@@ -1,143 +1,138 @@
-    each branch). If this parameter is omitted, the code gets the default value
-    (0.6).
--   `-sink_clustering_enable` enables pre-clustering of sinks to create one
-    level of sub-tree before building H-tree. Each cluster is driven by buffer
-    which becomes end point of H-tree structure.
--   `-sink_clustering_size` specifies the maximum number of sinks per
-    cluster. Default value is 20.
--   `sink_clustering_max_diameter` specifies maximum diameter (in micron)
-    of sink cluster. Default value is 50.
--   `-clk_nets` is a string containing the names of the clock roots. If
-    this parameter is omitted, TritonCTS looks for the clock roots automatically.
-
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-buf_list` | Tcl list of master cells (buffers) that will be considered when making the wire segments (e.g. `{BUFXX, BUFYY}`). |
+| `-root_buffer` | The master cell of the buffer that serves as root for the clock tree. If this parameter is omitted, the first master cell from `-buf_list` is taken. |
+| `-wire_unit` | Minimum unit distance between buffers for a specific wire. If this parameter is omitted, the code gets the value from ten times the height of `-root_buffer`. |
+| `-clk_nets` | String containing the names of the clock roots. If this parameter is omitted, TritonCTS looks for the clock roots automatically. |
+| `-distance_between_buffers` | Distance (in microns) between buffers that TritonCTS should use when creating the tree. When using this parameter, the clock tree algorithm is simplified, and only uses a fraction of the segments from the LUT. |
+| `-branching_point_buffers_distance` | Distance (in microns) that a branch has to have in order for a buffer to be inserted on a branch end-point. This requires the `-distance_between_buffers` value to be set. |
+| `-clustering_exponent` | Value that determines the power used on the difference between sink and means on the CKMeans clustering algorithm. (default 4, integer). |
+| `-clustering_unbalance_ratio` | Value that determines the maximum capacity of each cluster during CKMeans. A value of 50% means that each cluster will have exactly half of all sinks for a specific region (half for each branch). (default 0.6, float). |
+| `-sink_clustering_enable` | Enables pre-clustering of sinks to create one level of sub-tree before building H-tree. Each cluster is driven by buffer which becomes end point of H-tree structure. |
+| `-sink_clustering_size` | Specifies the maximum number of sinks per cluster. (default 20, integer). |
+| `sink_clustering_max_diameter` | Specifies maximum diameter (in microns) of sink cluster (default 50, integer). |
+| `-clk_nets` | String containing the names of the clock roots. If this parameter is omitted, TritonCTS looks for the clock roots automatically. |
 
 ### Report CTS
 
 Another command available from TritonCTS is `report_cts`. It is used to
-extract metrics after a successful `clock_tree_synthesis` run. These
```

Comment:
As discussed, the same of the original work should be mentioned in the first paragraph of the readme. In all other places we should refer to the OR module name, in this case `cts`.